diagnostic.md

```diff
@@ -150,7 +179,10 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne - on the right
+divTwo - left
+divThree - left
+divFour - right
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).